packages/kbn-interpreter/src/plugin/types/kibana_table.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const kibanaTable = () => ({
+  name: 'kibana_table',
+  serialize: context => {
+    context.columns.forEach(column => {
+      column.aggConfig = column.aggConfig.toJSON();
+    });
+    return context;
+  },
+  deserialize: () => {
+    // config.visData.columns.forEach(column => {
+    //   column.aggConfig = new AggConfig(a, column.aggConfig);
+    // });
+  },
+  validate: tabify => {
+    if (!tabify.columns) {
+      throw new Error('tabify must have a columns array, even if it is empty');
+    }
+  },
+  from: {
+    null: () => {
+      return {
+        type: 'kibana_table',
+        columns: [],
+      };
+    },
+    datatable: context => {
+      const converted = {
+        columns: context.columns.map(column => {
+          return {
+            id: column.name,
+            title: column.name,
+            ...column
+          };
+        }),
+        rows: context.rows.map(row => {
+          const crow = {};
+          context.columns.forEach(column => {
+            crow[column.name] = (row[column.name]);
+          });
+          return crow;
+        })
+      };
+      return {
+        type: 'kibana_table',
+        ...converted,
+      };
+    },
+    number: context => {
+      return {
+        type: 'kibana_table',
+        columns: [{ id: 'col-0', title: 'Count' }],
```

Comment:
I would suggest we name this `Value` instead, since we don't know what that number represent. It's just a numeric value, so I think `Value` would be the better fitting title here.